features/profiler.yml

```
@@ -1,7 +1,7 @@
-draft_date: 2024-08-19
-name: JS Self-Profiling API
-description: TODO
+name: Profiler
+description: The `Profiler` API traces and records JavaScript execution on a page, which you can send to a server for later analysis.
```

I like going with _Profiler_ as the name of the feature. That said, I think we should acknowledge the existence of the weird spec title. 😄 

```suggestion
description: The `Profiler` API traces and records JavaScript execution on a page, which you can send to a server for later analysis. Also known as self-profiling.
```